### DISCUSSION TITLE
Register Indexer

### DISCUSSION THREAD

**Discussion by zhephirothz** [subquery/subql#1052] [Frontier TestNet - Bug Reporting]
After I unregister.
I can't do a register even I use the same data fill&identity before I unregister.

Dev console shown its can't connect to my indexer ip 400(Bad Request)
* Try to use a ip number but didn't work also

![Screen Shot 2022-05-22 at 1 54 46 AM](https://user-images.githubusercontent.com/20436144/169665544-eb65d6bf-e98e-46fb-b1db-9586006ecf01.png)

**Comment by mzxyz** [comment]
Try to use another account to reregister
